src/index.ts

```diff
@@ -49,10 +49,18 @@ class TranscriptionEditor {
      * @param {any} storage Storage plugin to save Annotorious annotations.
      * @param {HTMLElement} annotationContainer Existing HTML element that the editor will be
      * placed into.
+     * @param {string} directionality Direction of the transcription content.
```

Comment:
might be clearer to clarify this is text direction, at least in the docstring. I guess you're exposing a tiny editor config and that's what they call it?

Reply:
That's right—clarified by name and comment and made optional, RTL by default.